specification/webpubsub/data-plane/WebPubSub/stable/2021-10-01/webpubsub.json

```diff
@@ -0,0 +1,1515 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Web PubSub Service REST API",
+    "version": "2021-10-01"
+  },
+  "paths": {
+    "/api/health": {
+      "head": {
+        "tags": [
+          "general"
+        ],
+        "summary": "Get service health status.",
+        "operationId": "HealthApi_GetServiceStatus",
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "api-version",
+            "description": "The version of the REST APIs.",
+            "required": true,
+            "type": "string",
+            "default": "2021-10-01"
```

Comment:
`required` + `default` is not allowed in openapi. This should be a single-valued enum value to indicate that it can only be a single value.

Reply:
removed `default`, sounds enough?